plugins/net_plugin/include/eosio/net_plugin/net_utils.hpp

```diff
@@ -51,18 +51,19 @@ namespace detail {
    }
 
    /// @return host, port, remainder
-   inline std::tuple<std::string, std::string, std::string> split_host_port_remainder(const std::string& peer_add, bool should_throw) {
+   inline std::tuple<std::string, std::string, std::string> split_host_port_remainder(const std::string& peer_add_input, bool should_throw) {
```

Comment:
Not from this PR. It would be more clearer if those `add`s are `addr`s. `add` sounds like the verb "add". Not need to change. Just a thought.

Reply:
Renamed